### PR TITLE
feat: add kc script to migrate/import idir users

### PR DESCRIPTION
**Pull request by junminahn**
- kc script to migrate one `IDIR` parent realm users to another.
- kc script to import `IDIR` users from parent realm to standard realm.

### CODE REVIEW

File: helm/keycloak/values-b861c7-test-4.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+image:
+  tag: 7.5-9-rc.2
+
+service:
+  type: ClusterIP
+  port: 8443
+
+postgres:
+  host: sso-patroni
+  database: ssokeycloak4
```
Comment (thegentlemanphysicist) [inline helm/keycloak/values-b861c7-test-4.yaml:16]: Are these all getting numbered so they can stay in the same patroni cluster

File: scripts/keycloak-import-idir-users.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      total += count;
+
+      for (let x = 0; x < users.length; x++) {
+        const { id, username, attributes, email, firstName, lastName } = users[x];
+
+        console.log(username);
+
+        // 1. delete the user with the username if already exists
+        const duplicates = await adminClient.users.find({ realm, username: `${username}@${IDP_REALM}`, max: 1 });
+        await sleep(200);
```
Comment (thegentlemanphysicist) [inline scripts/keycloak-import-idir-users.js:48]: why do we have a sleep here? won't the away pause things until the find resolves?

**Review by thegentlemanphysicist — COMMENTED** [review]
Left a couple of comments.  More generally, I'm missing context on how this script is run, and what kind of authentication is needed to run these?  Are these just needing the other credentials set up in the .env?

File: helm/keycloak/values-b861c7-test-4.yaml (hunk 1 above)
Comment (junmin-bcgov) [reply helm/keycloak/values-b861c7-test-4.yaml:16]: yes, as all sandbox environments in silver and gold are being used for other purposes, I deployed them, all 6, into a single environment to have them intact.

Comment (junmin-bcgov) [reply helm/keycloak/values-b861c7-test-4.yaml:16]: they all 6 shared one patroni cluster creating a separate db

File: scripts/keycloak-import-idir-users.js (hunk 2 above)
Comment (junmin-bcgov) [reply scripts/keycloak-import-idir-users.js:48]: right. I put them a couple of places to prevent the script gets timed out.

**Review by thegentlemanphysicist — APPROVED** [review]

**Pull request merged**